### PR TITLE
Add .idea/ to .gitignore

### PR DESCRIPTION
**Pull request by SimonBaeumer**
Signed-off-by: Simon Baeumer <sbaumer@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (lavalamp) [inline .gitignore:1]: Please add a comment to explain what this is (see format of https://github.com/kubernetes/kubernetes/blob/master/.gitignore).

Comment (SimonBaeumer) [reply .gitignore]: Copied over all IDE related ignore

Comment (SimonBaeumer) [reply .gitignore]: Thanks for the review.
Copied over all IDE related ignore config from the kubernetes repo and OSX files.

**Pull request merged**